static-site/src/components/ListResources/useSortAndFilter.ts

```diff
@@ -1,30 +1,39 @@
 import { useMemo } from 'react';
+import { FilterOption, Group, Resource } from './types';
 
 
-export const useSortAndFilter = (sortMethod, selectedFilterOptions, originalData, setState) => {
+export const useSortAndFilter = (sortMethod: string, selectedFilterOptions: readonly FilterOption[], setState: React.Dispatch<React.SetStateAction<Group[]>>, originalData?: Group[]) => {
```

Comment:
```
export const useSortAndFilter = (sortMethod: string, selectedFilterOptions: readonly FilterOption[], setState: React.Dispatch<React.SetStateAction<Group[]>>, originalData?: Group[]) => {
```

Since `sortMethod` here is being used in a `switch`, it might be worth typing it explicitly as `"lastUpdated"|"alphabetical"`, and then you can remove the `default` case.

Reply:
Updated in [force-push](https://github.com/nextstrain/nextstrain.org/compare/5d85bd9d18b175c0d7a304960eccbce53e411f82..1ecff3bd305af2b9942c6d8d9b1283e6ff80a8a2).

What do you think about using a type alias `SortMethod = "lastUpdated" | "alphabetical"`? Example: 3284beb526722968cdac41dadbbe0b091b82bd57

In the end, at least without more involved code changes, it requires a cast `event.currentTarget.value as SortMethod`, but I think it's a good improvement.

Reply:
yeah, i like making the type alias; it makes it more explicit that the two different places it is used are actually the same thing. 

agreed on the cast as a necessary evil in this case.

Reply:
Added the type alias to 6d3bf4f9cdc3a9a7d6c830068b34b0894895ed3a in [force-push](https://github.com/nextstrain/nextstrain.org/compare/1ecff3bd305af2b9942c6d8d9b1283e6ff80a8a2..fb2fb81020eaca1644dadc9d3041d6120c178a7d).